ParisBNF/abb/BNFabb28.xml

```diff
@@ -0,0 +1,312 @@
+                        <collation>
+                          <note>The structure of the quires should be ascertained directly on the physical object.</note>
+                        </collation>
+                        <condition key="good">Half of the leaf <locus target="#70"/> and the bottom margin of <locus target="#77"/> are cut; 
+                           the leaf <locus target="#88"/> is mutilated. Two notes on <locus target="#1v"/> are blotted out, and one note erased on <locus target="#54r #75 #84r"/> . 
+                           Some leaves are torn and repaired (<locus target="#5 #36"/>).</condition>
+                     </supportDesc>
+
+                     <layoutDesc>
+                       <layout columns="2" writtenLines="28 30">
+                            <locus from="2r" to="86vb"/>
+                          <note>A small number of lines (25) on <locus from="87" to="88"/>.</note>
+                        <ab type="pricking">Pricking is partly visible.</ab>
+                          <ab type="ruling">Ruling is not visible.</ab>
+                        <ab type="ruling" subtype="pattern">1A-1A-1A1A/0-0/0-0/C</ab>
+                          </layout>
+                     </layoutDesc>
+
+                  </objectDesc>
+                  <handDesc>
+                   <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="script">18th-century script, an earlier period (17th century) cannot be excluded.</seg>
+                             <seg type="ink">Black, red.</seg>
+                             <desc>Careful, somewhat irregular; by a good scribe.
+                               The script is small, somewhat difficult to evaluate (due to the insufficient 
+                               quality of the images). Small blank spaces are left on many folia, for unclear purpose.</desc>
+                             </handNote>
+                  </handDesc>
+                  <additions>
+                    <list>
+                      <item xml:id="a1">
+                         <locus target="#1v"/>
+                         <desc type="Unclear">An unclear note in the upper left part of the leaf, written probably in a language other than Gǝʿǝz or Amharic</desc>
+                            </item>
+                                        
+                       <item xml:id="a2">
+                          <locus target="#1v"/>
+                          <desc type="Unclear">An unclear note, possibly referring to an owner of the book</desc>
+                          <q xml:lang="gez">አምላክ፡ ለኵሉ፡ መንፈስ፡ ወለኵሉ፡ ዘሥጋ፡ ዝመጽሐፍ፡ ዘወንድም፡ የመጻፍ፡ ስሙ፡ ቤዝወኒ፡</q>
+                       </item>
+                       
+                       <item xml:id="a3">
+                          <locus target="#2r"/>
+                          <desc type="Unclear">A blessing formula suggestive of a relation to Beta ʾƎsrāʾel; most probably added later, in a different hand 
```

Comment:
`<persName ref="ETH1274Betaes">Beta ʾƎsrāʾel</persName>`

Reply:
I can do; my recent experience was that encoding in this specific place (as within `<title/>`) will not be visualised.

Reply:
OK